### DISCUSSION TITLE
ISSkioM2019 problems with tracks on top

### DISCUSSION THREAD

**Discussion by kjetilk** [OpenOrienteering/mapper#2202] [Questions & Answers] [answered]
I'm struggling a bit with a ski-o map I'm making. I made the map by stitching together an ISOM2000 and a ISSprOM 2019-2 map and applying a ISOM - ISSkioM CRT, and editing a bit manually. It was a little bit messy, but I think the maps looks pretty nice now -- fit for purpose.

I'm struggling with tracks themselves. The track symbols are numbered 801-804, and now they go underneath pretty much everything else. For example, I have lot of 403 Open Area symbols on the stadium, and the tracks go underneath that, so they can't be seen. How do I fix that? 

I suppose I could make the rest of the map a template? There are two reasons why I haven't done that yet, one is that my plan was to switch many of the paths and roads to tracks, that would make for better readability and easier editing. Furthermore, it is my plan to use purplepen for the actual event, and then I wasn't sure how it would work if I used the templating system.

So, my original plan was to put all the roads and paths to be switched to a part of their own, but that doesn't seem doable now.

What would your suggestion be?

**Comment by Choreuomai** [comment, marked as answer]
Hi, from what you write about the tracks I would suggest you simply look at which colour is used for the tracks and change it and/or adapt the order in the colour table. But maybe I didnt understand the problem without seeing the map.

Am 25. Januar 2024 02:12:52 MEZ schrieb Kjetil Kjernsmo ***@***.***>:
>I'm struggling a bit with a ski-o map I'm making. I made the map by stitching together an ISOM2000 and a ISSprOM 2019-2 map and applying a ISOM - ISSkioM CRT, and editing a bit manually. It was a little bit messy, but I think the maps looks pretty nice now -- fit for purpose.
>
>I'm struggling with tracks themselves. The track symbols are numbered 801-804, and now they go underneath pretty much everything else. For example, I have lot of 403 Open Area symbols on the stadium, and the tracks go underneath that, so they can't be seen. How do I fix that? 
>
>I suppose I could make the rest of the map a template? There are two reasons why I haven't done that yet, one is that my plan was to switch many of the paths and roads to tracks, that would make for better readability and easier editing. Furthermore, it is my plan to use purplepen for the actual event, and then I wasn't sure how it would work if I used the templating system.
>
>So, my original plan was to put all the roads and paths to be switched to a part of their own, but that doesn't seem doable now.
>
>What would your suggestion be?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/OpenOrienteering/mapper/discussions/2202
>You are receiving this because you are subscribed to this thread.
>
>Message ID: ***@***.***>
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Reply by kjetilk** [reply]
Ah, I was of the mistaken belief that the order of the symbols was fixed from the symbol set. I didn't realize I could trivially change it, but indeed, that was easy to spot once I understood it could be done. Thanks a lot!
